frontend/components/MainPage/MainHeader/FirstRunModal.tsx

```diff
@@ -3,18 +3,16 @@ import Image from 'next/image';
 import { FC } from 'react';
 
 import { MODAL_WIDTH } from '@/constants/width';
-import { useServiceTemplates } from '@/hooks/useServiceTemplates';
+import { TokenSymbol } from '@/enums/Token';
+import { useStakingProgram } from '@/hooks/useStakingProgram';
 
 type FirstRunModalProps = { open: boolean; onClose: () => void };
 
 export const FirstRunModal: FC<FirstRunModalProps> = ({ open, onClose }) => {
-  const { getServiceTemplates } = useServiceTemplates();
+  const { activeStakingProgramMeta } = useStakingProgram();
 
-  if (!open) return null;
-
-  const minimumStakedAmountRequired = getMinimumStakedAmountRequired(
-    getServiceTemplates()[0],
-  );
+  const minimumStakedAmountRequired =
```

Comment:
I'm now confused how come it's not here https://github.com/valory-xyz/olas-operate-app/pull/484/files#diff-6faea1404e840192e28685e7a19f40d301a77f8368b864476e1e0131defed7c3


oh, those are duplicates 😱